### PR TITLE
[7976]  wagtail 5.2.x

### PR DESCRIPTION
**Pull request by goapunk**
probably needs the django update first

### CODE REVIEW

File: tempelhof/settings/dev.py [hunk 1]
```diff
@@ -3,6 +3,8 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
+WAGTAILADMIN_BASE_URL = "http://127.0.01:8007"
```
Comment (m4ra) [inline tempelhof/settings/dev.py:6]: this is needed for the production too. are u adding it in the admin repo? I see the warning from django upgrade

> ?: (wagtailadmin.W003) The WAGTAILADMIN_BASE_URL setting is not defined
> 	HINT: This should be the base URL used to access the Wagtail admin site. Without this, URLs in notification emails will not display correctly.

Comment (m4ra) [reply tempelhof/settings/dev.py:6]: it would be good to switch to python 3.10 also

Comment (goapunk) [reply tempelhof/settings/dev.py:6]: ubuntu 20.04 only packages 3.8 or 3.9 I think, so we'd have to upgrade to 24.04. I guess it's a good way to start as it's a rather small project

**Comment by goapunk** [comment]
closing as superseded by #545

**Pull request closed without merging**